vpr/src/draw/draw.cpp

```diff
@@ -152,6 +155,7 @@ void run_graphics_commands(std::string commands);
 constexpr float SB_EDGE_TURN_ARROW_POSITION = 0.2;
 constexpr float SB_EDGE_STRAIGHT_ARROW_POSITION = 0.95;
 constexpr float EMPTY_BLOCK_LIGHTEN_FACTOR = 0.20;
+float net_alpha = 0.0275;
```

Comment:
We should move `net_alpha` to be a member of t_draw_state, rather than a file-scope variable.

That should ensure that if the draw state is reset, so is the net alpha.

Reply:
We should also probably modify the default to be a little less transparent (e.g. 0.1), as with 0.0275 single connections are very faint.